mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala

```diff
@@ -72,7 +72,7 @@ class MySQLMetadataManager(hostname: String, port: Int, username: String, passwo
       val mapCols = getTableColumns(db, table, dbConn(0))
```

Comment:
on line 68, database connection times out after 5 seconds. Eric and I think it's generous enough, but open to changing it to be configurable as well

Reply:
We're seeing this timing out on invalidators, unfortunately (see https://app.scalyr.com/events?filter=message+contains+%22getTableColumns%22&startTime=7d&teamToken=CoX7TpFAqLjqYwbafYKNKQ--&log=%2Fvar%2Flog%2Fworldst-prod_invalidator%2Finvalidator.log for both the 1 second and the 5 second timeouts)

Reply:
@arvindvjkumar  Just made this configurable as well, do you mind reviewing and testing locally again?

Reply:
I'll bump this higher + validate test in sand later today